Cargo.toml

```diff
@@ -33,9 +33,9 @@ num-traits = "0.2.15"
 once_cell = "1"
 rand = "0.8.5"
 reqwest = { version = "0.11.13", features = ["json"] }
-schemars = "0.8"
+schemars = "0.8.12"
```

Comment:
Hi, I had issues in 0.8 so I had 0.8.13 in my PR, maybe you can try 13 too ? thanks